src/Tizen.NUI.Components/Controls/ItemsView/ItemsView.cs

```diff
@@ -0,0 +1,454 @@
+    /// </summary>
+    /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class ItemsView : ScrollableBase, ICollectionChangedNotifier
+    {
+        private List<PropertyNotification> notifications = new List<PropertyNotification>();
+        /// <summary>
+        /// Base Constructor
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        public ItemsView() : base()
+        {
+            Scrolling += OnScrolling;
+        }
+
+        /// <summary>
+        /// Base Constructor with itemsSource
+        /// </summary>
+        /// <param name="itemsSource">item's data source</param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        public ItemsView(IEnumerable itemsSource) : base()
+        {
+            ItemsSource = itemsSource;
+        }
+
+        /// <summary>
+        /// Base Constructor with itemsSource
+        /// </summary>
+        /// <param name="itemsSource">item's data source</param>
+        /// <param name="layouter">item's layout manager</param>
+        /// <param name="template">item's view template with data bindings</param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        public ItemsView(IEnumerable itemsSource, ItemsLayouter layouter, DataTemplate template) : base()
+        {
+            ItemsSource = itemsSource;
+            ItemsTemplate = template;
+            ItemsLayouter = layouter;
+        }
+
+        /// <summary>
+        /// Scrolling Direction.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        public new Direction ScrollingDirection
+        {
+            get
+            {
+                return base.ScrollingDirection;
+            }
+            set
+            {
+                base.ScrollingDirection = value;
+            }
+        }
+
+        private IEnumerable _itemsSource = null;
+        /// <summary>
+        /// Item's source data.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IEnumerable ItemsSource
+        {
+            get
+            {
+                return _itemsSource;
+            }
+            set
+            {
+                _itemsSource = value;
+            }
+        }
```

Comment:
`IDE0032: Use auto property`
```suggestion
        /// <summary>
        /// Item's source data.
        /// </summary>
        /// <since_tizen> 8 </since_tizen>
        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
        [EditorBrowsable(EditorBrowsableState.Never)]
        public IEnumerable ItemsSource { get; set; } = null;
```